### PR TITLE
ucms1_typos1

### PR DESCRIPTION
**Pull request by shlff**
Good day, @jstac , this PR corrects following typos in lecture [us_mc_semigroup](https://github.com/jstac/continuous_time_mcs/blob/master/ctmc_lectures/uc_mc_semigroups.md):
- ``matricies`` -> ``matrices``,
- ``charcterization`` -> ``characterization``,
- ``a intensity matrix`` -> ``an intensity matrix``,
- ``first along the principle diagonal`` -> ``first along with the principle diagonal``,
- ``contably infinite`` -> ``countably infinite``

### CODE REVIEW

File: ctmc_lectures/uc_mc_semigroups.md [hunk 1]
```diff
@@ -360,14 +360,14 @@ As we now show, every intensity matrix admits the decomposition in
...
-Given a intensity matrix $Q$, set 
+Given an intensity matrix $Q$, set 
 
 $$
     \lambda(x) := -Q(x, x)
     \qquad (x \in S)
 $$ (lambdafromq)
 
-Next we build $K$, first along the principle diagonal via
+Next we build $K$, first along with the principle diagonal via
```
Comment (jstac) [inline ctmc_lectures/uc_mc_semigroups.md:370]: Please undo this change.

**Comment by jstac** [comment]
Thanks @shlff, this is helpful.  I just requested one small change ("with" is not necessary).

**Comment by shlff** [comment]
Thanks, @jstac . I only undo the "with" one and keep all other changes.

**Comment by jstac** [comment]
Thanks @shlff .
